app/__init__.py

```diff
@@ -25,4 +25,9 @@ def create_app(config_name):
     from .main import main as main_blueprint
     app.register_blueprint(main_blueprint)
 
+    """ auth blueprint """
+    from .auth import auth as auth_blueprint
+    app.register_blueprint(auth_blueprint)
```

Comment:
Looks good. I would suggest you add an `url_prefix` to prevent routes collision in the future

```
app.register_blueprint(auth_blueprint, url_prefix='/auth')
```